x-pack/plugins/monitoring/server/kibana_monitoring/collectors/get_usage_collector.ts

```diff
@@ -18,9 +18,9 @@ import { fetchClusters } from '../../lib/alerts/fetch_clusters';
 export function getMonitoringUsageCollector(
   usageCollection: UsageCollectionSetup,
   config: MonitoringConfig,
-  callCluster: LegacyAPICaller
+  esClient: ILegacyClusterClient
```

Comment:
This feels wrong in terms of nomenclature. `esClient` _should_ indicate we're using the new elasticsearch client, whereas `callCluster` specifically refers to the legacy elasticsearch client. When I first dug into the issue of the `esClient` being "missing" for the SO tags collector issue, I thought it was because the new elasticsearch client wasn't available yet. Even so, the legacy client should be available from the `setup` contract, so how are we solging for the client not be available yet?